docs/resources/cloud_project_database_log_subscription.md

```diff
@@ -27,6 +27,7 @@ resource "ovh_cloud_project_database_log_subscription" "subscription" {
 	engine       = data.ovh_cloud_project_database.db.engine
 	cluster_id   = data.ovh_cloud_project_database.db.id
 	stream_id    = data.ovh_dbaas_logs_output_graylog_stream.stream.id
+  kind         = "customer_logs"
```

Comment:
you should also edit:
- https://github.com/ovh/terraform-provider-ovh/blob/master/examples/resources/cloud_project_database_log_subscription/example_1.tf
- https://github.com/ovh/terraform-provider-ovh/blob/master/templates/resources/cloud_project_database_log_subscription.md.tmpl

as the doc is partly generated now.

Also, could you align this line with the ones above ?